php/post-types/class-block-post.php

```diff
@@ -24,7 +24,7 @@ class Block_Post extends Component_Abstract {
 	 *
 	 * @var string
 	 */
-	public $slug = 'acb_block';
+	public $slug = 'block_lab_block';
```

Comment:
This slug causes a lot of weird class names and IDs. I can't think of a better option though. `bl` is too short, and it's inconsistent. Is `custom_block` too generic?

Reply:
I think custom_block works. It's descriptive.
However, being so generic, is there risk of conflict with other plugins/themes?

Reply:
Yes, if there's a theme or plugin that also uses that name then there would be a conflict.

Reply:
Another (more generic) option is just `block`. What's the chance someone will want to use that CPT name in their theme or plugin?

Reply:
I'm not 100% on the use case of the slug and how it's used, so my only insight is that the more generic the slug the more likely it will be used.

Reply:
Another option could be lab_block

Reply:
Yeah Block Lab doesn't lend itself to nice shortening.  As this is a Post Type, why not just call it `block_lab` for the slug?  I know its not very descriptive, but where else will this post type actually be used?